### PR TITLE
13152 Auth Web - Marketing Page Updates

### PR DESCRIPTION
**Pull request by lewischenstudio**
*Issue #:* https://github.com/bcgov/entity/issues/13152

*Description of changes:*
- added new look to "Register or Incorporate" component 
- added unit tests
- fixed typo
- design UI link: [here](https://projects.invisionapp.com/share/5C12X9U7RUJT#/screens/468822175) and [here](https://projects.invisionapp.com/share/5C12X9U7RUJT#/screens/468822176)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/tests/unit/views/RequestNameView.spec.ts [hunk 1]
```diff
@@ -66,18 +66,17 @@ describe('RequestNameView.vue', () => {
...
+      { text: 'Bullet Mock 2' }
     ]
 
     const bulletList = wrapper.vm.$el.querySelectorAll('.list-item')
 
     expect(bulletList[0].textContent).toContain('Bullet Mock 1')
     expect(bulletList[1].textContent).toContain('Bullet Mock 2')
-    expect(bulletList[2].textContent).toContain('Bullet Mock 3')
-    expect(bulletList[3].textContent).toContain('You can choose to incorporate')
+    expect(bulletList[2].textContent).toContain('Submit your name choices for examniation')
```
Comment (seeker25) [inline auth-web/tests/unit/views/RequestNameView.spec.ts:76]: Typo - examniation

**Review by seeker25 — APPROVED** [review]
Looks good, maybe some minor text changes.

**Comment by seeker25** [comment]
Code smell as well `      <!-- </v-list> -->`

File: auth-web/src/views/auth/home/HomeView.vue [hunk 2]
```diff
@@ -29,7 +29,7 @@
                     <v-btn
                       large
                       color="bcgovblue"
-                      class="hero-banner-login-btn mr-4 font-weight-bold white--text"
```
[the remark this answers is not in the document]
Comment (lewischenstudio) [reply auth-web/src/views/auth/home/HomeView.vue:32]: 👍 The css was removed in [Alex's PR](https://github.com/bcgov/sbc-auth/pull/2060) this morning, so it should be good to remove the class too.

File: auth-web/src/views/auth/home/IncorpOrRegisterView.vue [hunk 3]
```diff
@@ -66,10 +94,29 @@ import { appendAccountId } from 'sbc-common-components/src/util/common-util'
 export default class IncorpOrRegisterView extends Vue {
   protected readonly learnMoreUrl = 'https://www2.gov.bc.ca/gov/content/governments/organizational-structure/ministries-organizations/ministries/citizens-services/bc-registries-online-services'
   private readonly bulletPoints: Array<any> = [
-    { text: 'For Named Companies, add your existing Name Request number to your account and open it.' },
-    { text: 'Establish your company\'s articles and prepare an Incorporation Agreement. Either create your own, or use template provided in the Incorporation Application.' },
-    { text: 'Complete the Incorporation Application by providing information about your company: addresses, directors and share structure.' },
-    { text: 'Retain a copy of all Incorporation documents for your business\'s records.' }
+    { text: 'Register a firm such as sole proprietorship, DBA, general partnership, and societies' },
```
[the remark this answers is not in the document]
Comment (lewischenstudio) [reply auth-web/src/views/auth/home/IncorpOrRegisterView.vue:97]: no periods in the design: [here](https://projects.invisionapp.com/share/5C12X9U7RUJT#/screens/468822176)

File: auth-web/tests/unit/views/RequestNameView.spec.ts (hunk 1 above)
Comment (lewischenstudio) [reply auth-web/tests/unit/views/RequestNameView.spec.ts]: 👍 Good catch. It's now fixed.

File: auth-web/src/views/auth/home/IncorpOrRegisterView.vue [hunk 4]
```diff
@@ -66,10 +94,29 @@ import { appendAccountId } from 'sbc-common-components/src/util/common-util'
...
+    { text: 'Incorporate a benefit company or a cooperative association' }
+  ]
+
+  private readonly expansionPanels: Array<any> = [
+    { text: 'Sole Proprietorship, DBA, and General Partnership',
```
Comment (shabeeb-aot) [inline auth-web/src/views/auth/home/IncorpOrRegisterView.vue:102]: en.json is another good place to keep text

**Review by shabeeb-aot — APPROVED** [review]

File: auth-web/src/views/auth/home/IncorpOrRegisterView.vue [hunk 5]
```diff
@@ -3,25 +3,53 @@
...
+                <v-icon size="8" class="list-item-bullet mt-5">mdi-square</v-icon>
+                <v-list-item-content>
+                  <v-list-item-subtitle class="list-item-text">
+                    {{item.text}}
+                  </v-list-item-subtitle>
+                </v-list-item-content>
+              </v-list-item>
             </v-list-item-content>
           </v-list-item>
+        <!-- </v-list> -->
```
Comment (shabeeb-aot) [inline auth-web/src/views/auth/home/IncorpOrRegisterView.vue:25]: remove commented code

Comment (lewischenstudio) [reply auth-web/src/views/auth/home/IncorpOrRegisterView.vue]: Done.

File: auth-web/src/views/auth/home/IncorpOrRegisterView.vue (hunk 4 above)
Comment (lewischenstudio) [reply auth-web/src/views/auth/home/IncorpOrRegisterView.vue:101]: Hmm. It doesn't look like a good place for this array style text. I'll keep it in mind though.

**Comment by seeker25** [comment]
Yup, I'll get it up on test for you in about 15 minutes

**Comment by seeker25** [comment]
Should be on test

**Comment by lewischenstudio** [comment]
Thanks a lot for the merge/push.

**Pull request merged**